lib/inets/test/httpc_SUITE.erl

```diff
@@ -569,7 +569,7 @@ async(Config) when is_list(Config) ->
 	    {http, Msg} ->
 		ct:fail(Msg)
 	end,
-    inets_test_lib:check_body(binary_to_list(Body)),
+    inets_test_lib:check_body(Body),
```

Comment:
should we have some new testcase or updates in some of the existing?

Reply:
probably testcase for checking full_result option would be viable